server/webapp/WEB-INF/rails.new/app/views/navigation_elements/_dropdown_template_admin.html.erb

```diff
@@ -0,0 +1,3 @@
+<li role="presentation">
+  <%= link_to l.string("TEMPLATES"), main_app.templates_path -%>
+</li>
```

Comment:
Do we really need a partial for 3 lines of a link that's used in just one place? Remember that partials have a rendering overhead.